CPP/Algorithm/tower_of_hanoi.cpp

```diff
@@ -0,0 +1,28 @@
+// C++ recursive function to  
+// solve tower of hanoi puzzle  
+#include <bits/stdc++.h> 
+using namespace std; 
+  
+void towerOfHanoi(int n, char from_rod, 
```

Comment:
I guess you didn't read the existing code. Please have a look at existing c++ code and try to follow the same coding style. These canges are not in accordance with the existing one.